react/ProductSummaryImage.tsx

```diff
@@ -183,12 +192,19 @@ function BadgeWrapper({
 
 function findImageByLabel(
   images: ProductSummaryTypes.SKU['images'],
-  selectedLabel: string | undefined
+  selectedLabel: string | undefined,
+  labelMatchCriteria: ImageLabelMatchCriteria | undefined = 'exact'
 ) {
   if (!selectedLabel) {
     return null
   }
 
+  if (labelMatchCriteria === 'contains') {
+    return images.find(
+      ({ imageLabel }) => imageLabel?.indexOf(selectedLabel) !== -1
```

Comment:
Just a suggestion
```suggestion
      ({ imageLabel }) => imageLabel?.includes(selectedLabel)
```

Reply:
@igorbrasileiro Great suggestion, thank you!